modules/standard/Collection.chpl

```diff
@@ -0,0 +1,116 @@
+/*
+  A 'Collection' is a data structure, a container for elements that provide support
+  for insert, lookup, remove, and iteration operations.
+*/
+class Collection {
+  /*
+    The type of element that this Collection holds.
+  */
+  type eltType;
+
+  /*
+    Adds an element to this data structure.
+  */
+  inline proc add(elt : eltType) : bool {

```

Comment:
The reasoning behind declaring it `inline` is due so that the user will know the precisely location they have invoked the unsupported method. Normally, if it is not inlined, it shows a line number in Collections.chpl, which isn't entirely helpful.